MSAL/src/native_auth/network/MSALNativeAuthResponseSerializer.swift

```diff
@@ -40,7 +40,7 @@ final class MSALNativeAuthResponseSerializer<T: Decodable & MSALNativeAuthRespon
             response.correlationId = T.retrieveCorrelationIdFromHeaders(from: httpResponse)
```

Comment:
@diegojerezba could you also check the log  `MSALNativeAuthUrlRequestSerializer` at line 67? I think we should log that as PII because the request body can contains PII too.

Reply:
I decided not to add it because I think the exception thrown only contains the key. But I'm adding it just in case.